### PR TITLE
cds data added to search dumps; translation key added to search dumps

### PR DESCRIPTION
**Pull request by vinay-ebi**
CDS data added to search dumps (ENSPROD-5397)
initialise translation key  for all non-coding genes in  search dumps  (ENSPROD-5872)

### CODE REVIEW

File: modules/Bio/EnsEMBL/Production/DBSQL/BulkFetcher.pm [hunk 1]
```diff
@@ -390,6 +415,24 @@ sub get_transcripts {
     return $transcript_hash;
 } ## end sub get_transcripts
 
+
+sub set_cds {
+
+  my ($self , $exons_list, $transcript, $transcript_id ) = @_;
+
+  $log->debug("Getting CDS Data");
```
Comment (marcoooo) [inline modules/Bio/EnsEMBL/Production/DBSQL/BulkFetcher.pm:423]: Could you remove this line, please? Or update the log to be more "detailed".

**Review by marcoooo — CHANGES_REQUESTED** [review]
Another question, when testing, did you check the xml output after reformat for EBEYE? Has it changed?

File: modules/Bio/EnsEMBL/Production/DBSQL/BulkFetcher.pm (hunk 1 above)
Comment (vinay-ebi) [reply modules/Bio/EnsEMBL/Production/DBSQL/BulkFetcher.pm:423]: ok i will add detailed desc

Comment (vinay-ebi) [reply modules/Bio/EnsEMBL/Production/DBSQL/BulkFetcher.pm]: we need to explicitly mention in EBeyeFormatter.pm module for cds data changes to reflect the in xml.  @Kieron Taylor was concern about the json data.

Comment (marcoooo) [reply modules/Bio/EnsEMBL/Production/DBSQL/BulkFetcher.pm]: Ok so we don't change EBEYE format just yet. We'll ask Ebi search team about it, thanks.

File: modules/Bio/EnsEMBL/Production/DBSQL/BulkFetcher.pm [hunk 2]
```diff
@@ -390,6 +415,23 @@ sub get_transcripts {
     return $transcript_hash;
 } ## end sub get_transcripts
 
+
+sub set_cds {
+
+  my ($self , $exons_list, $transcript, $transcript_id ) = @_;
+
+  $log->debug("Getting CDS start and end cords for transcript");
```
Comment (marcoooo) [inline modules/Bio/EnsEMBL/Production/DBSQL/BulkFetcher.pm]: Not detailed enough I am afraid, can't you include in the log some actual data? 
Not to have this kind of log in the tests:
     
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data

**Review by marcoooo — APPROVED** [review]

**Pull request merged**